src/hooks/UseScrollBarRemove.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+
+interface ScrollBarRemoveProps {
+	isVisible: boolean;
+	children: React.ReactNode;
+}
+// useEffect(() => {
+// 	if (reviewForm) {
+// 		document.body.style.overflow = 'hidden';
+// 	} else {
+// 		document.body.style.overflow = 'auto';
+// 	}
+// 	return () => {
+// 		document.body.style.overflow = 'auto';
+// 	};
+// }, [reviewForm]);
+function UseScrollBarRemove({ isVisible, children }: ScrollBarRemoveProps) {
```

Comment:
remove this commented code, if you don't need it @Tuyisenge2

Reply:
it is removed @leandreAlly